dail/postponed-expedited-snap-verifications.vbs

```diff
@@ -104,13 +215,41 @@ DO
 		Dialog Dialog1
 		cancel_confirmation
 		Call validate_MAXIS_case_number(err_msg, "*")
-		IF verifs_needed = "" THEN err_msg = err_msg & vbCr & "* Please enter the postponed verifications requested."
-		IF worker_signature = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
+		IF trim(verifs_needed) = "" THEN err_msg = err_msg & vbCr & "* Please enter the postponed verifications requested."
+		IF len(trim(verifs_needed)) > 100 THEN err_msg = err_msg & vbCr & "* The list of verifications must be less than 100 characters long. Please shorten."
```

Comment:
What is the character limit for?

Reply:
Great question. Not sure, I think I misunderstood some direction with the script. I removed it.